src/api/core.rs

```diff
@@ -193,10 +165,7 @@ fn format_value_array_as_string(v: &[Value]) -> String {
     format!(
         "[{}]",
         v.iter()
-            .map(|elem| match elem {
-                v @ Value::String(_) | v @ Value::Bytes(_) => format!(r#""{}""#, String::from(v)),
-                v => String::from(v),
-            })
+            .map(String::from)
```

Comment:
I think the intention was to produce a JSON formatted array, which would require strings to be quoted. I also think this didn't produce correct JSON all the time because it didn't escape special characters in string values. But maybe we want to keep the quoted strings?

Reply:
Ah, good point, I skipped over that part a little. Fixed up in the next push.